courses/serializers.py

```diff
@@ -96,70 +97,139 @@ class Meta:
+            "prerequisites",
+        ]
+
     def get_prerequisites(self, instance):  # noqa: ARG002
-        """Get course prerequisites"""
+        """Get product prerequisites"""
 
         # This is a hard coded value because the consumers of the API need this field.
-        # In an ideal situation the prerequisites could be list of courses.
+        # In an ideal situation the prerequisites could be list of products.
         # Since in xPRO we don't have support for prerequisites
         # so we will not get/check for prerequisites
         return []
 
     def get_availability(self, instance):  # noqa: ARG002
-        """Get course availability"""
+        """Get product availability"""
 
         # This is a hard coded value because the consumers of the API need this field.
         # In an ideal situation the availability could be "dated" or "anytime".
-        # Since all the courses in xPRO are dated so we will not check for "self paced"
-        # courses to determine if the course could be "anytime"
+        # Since all the products in xPRO are dated so we will not check for "self paced"
+        # products to determine if the product could be "anytime"
         return "dated"
 
     def get_url(self, instance):
-        """Get CMS Page URL for the course"""
+        """Get URL"""
         page = instance.page
         return page.get_full_url() if page else None
 
+    def get_description(self, instance):
+        """Description"""
+        return instance.page.description if instance.page else None
+
     def get_external_marketing_url(self, instance):
-        """Return the external marketing URL for the course that's set in CMS page"""
+        """Return the external marketing URL for this product that's set in CMS page"""
         return instance.page.external_marketing_url if instance.page else None
 
     def get_marketing_hubspot_form_id(self, instance):
-        """Return the marketing HubSpot form ID associated with the course that's set in CMS page"""
+        """Return the marketing HubSpot form ID associated with the product that's set in CMS page"""
         return instance.page.marketing_hubspot_form_id if instance.page else None
 
     def get_thumbnail_url(self, instance):
         """Thumbnail URL"""
         return _get_thumbnail_url(instance.page)
 
+    def get_time_commitment(self, instance):
+        """Returns the time commitment for this product that's set in CMS page"""
+        return instance.page.time_commitment if instance.page else None
+
+    def get_duration(self, instance):
+        """Returns the duration for this product that's set in CMS page"""
+        return instance.page.duration if instance.page else None
+
+    def get_min_weeks(self, instance):
+        """
+        Get the min weeks of the product from the CMS page.
+        """
+        if hasattr(instance, "page") and hasattr(instance.page, "min_weeks"):
+            return instance.page.min_weeks
+
+        return None
```

Comment:
Why didn't we follow the same pattern of code which is used for other fields?

```suggestion
    def get_min_weeks(self, instance):
        """
        Get the min weeks of the product from the CMS page.
        """
         return instance.page.min_weeks if instance.page and instance.page.min_weeks else None
```